docker/package/scripts/tezos-baker-start

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+. "./tezos.inc.sh"
+
+shift
+
+launch_baker "$@"
```

Comment:
looks like baker daemon will require some additional config setup because when you start the baker you should provide baker address alias

Reply:
oh, nevermind, this could be done during the service start